scripts.py

```diff
@@ -1618,9 +1652,11 @@ def do_run(self):
         self._db.commit()
 
 
-class WorkPresentationScript(WorkProcessingScript):
+class WorkPresentationScript(TimestampScript, WorkProcessingScript):
     """Calculate the presentation for Work objects."""
 
+    name = "Recalculate the presentation for works that need it."""
```

Comment:
I didn't know you could use different numbers of quotes at the beginning and the end of a string.

Reply:
You can't really; this is equivalent to `"foo"bar" == "foobar"`. It works by accident.